releasenotes/notes/0.4-prelude-e12d8c852c35b7d4.yaml

```diff
@@ -0,0 +1,20 @@
+---
+prelude: >
+  Qiskit Finance has been migrated to the `qiskit-community Github organization <https://github.com/qiskit-community>`_
+  to further emphasize that it is a community-driven project.
+  To reflect this change and because we are on-boarding additional code owners and maintainers,
+  with this version (0.4) we have decided to *remove all deprecated* code, regardless of the time of its deprecation.
+  This ensures that the new members of the development team do not have a large bulk of legacy code to maintain.
+
+  This can mean one of two things for you as the end-user:
+
+    #. Nothing, if you already migrated your code and no longer rely on any deprecated features.
+
+    #. Otherwise, you need to migrate your code immediately. If you cannot do that, or want to continue using some
+       of the features that were removed, you should pin your version of Qiskit Finance to 0.3.4
+
+  You can check out the `migration guides <https://qiskit.org/ecosystem/optimization/migration/index.html>`_
+  for details on how to update your code.
```

Comment:
This is referring to the optimization migration guide. I guess in practice this really where the changes are though I think changes should be transparent to a user of Finance. Where the change from qiskit.algorithms to qiskit_algorithms was done was in applications here and in tutorials. Maybe we might add the note here that since Finance is dependent on Optimization, which now only supports Qiskit Algorithms (qiskit_algorithms), that you might like to refer to the optimization migration guide for further information in that regard.

Reply:
It's just a mistake. Because finance does not have migration page, I will remove this paragraph.